_internal/faq-migration-to-opensearch.md

```diff
@@ -0,0 +1,128 @@
+
+## How is backward compatibility maintained in OpenSearch?
+
+OpenSearch maintains backward compatibility by ensuring its APIs and core search functionality are compatible with Elasticsearch 7.10.2. When OpenSearch was forked from Elasticsearch 7.10.2, the goal was to keep the base functionality and APIs the same. This lets applications and tools built to work with Elasticsearch 7.10.2 to continue functioning with OpenSearch.
+
+Also, OpenSearch includes a compatibility mode, which lets OpenSearch respond with a version number of 7.10.2. This can be useful for tools or clients that check the version of the search engine and expect to communicate with Elasticsearch 7.10.2.
+
+However, while backward compatibility is a priority, certain features or functionalities from older versions of Elasticsearch are not supported in OpenSearch. We recommend  thoroughly testing your application in a non-production environment when upgrading or migrating to different software.
+
+Also, as OpenSearch continues to evolve, and new versions are released, for the updates on backward compatibility, make sure to check the [OpenSearch documentation](https://opensearch.org/docs/latest/) or their [GitHub repository](https://github.com/opensearch-project/OpenSearch).
+
+## Does OpenSearch support all the plugins that I used with Elasticsearch?
+
+OpenSearch supports most plugins compatible with Elasticsearch 7.10.2. However, make sure to check the compatibility of specific plugins in the OpenSearch documentation or with a particular plugin's provider. Some cases require plugins to be updated or replaced with OpenSearch-compatible versions. To get the list of the plugins you are currently using,  [contact our Support team](https://spryker.my.site.com/support/s/). Before the production environment is migrated, we will migrate your development and staging environments, so you can test backward compatibility.
+
+## How will the migration affect my configuration and settings?
+
+The migration from Elasticsearch to OpenSearch should not affect your current configuration and settings. OpenSearch is backward compatible with Elasticsearch 7.10.2, so it should respect your existing configuration and settings. However, when updating to a major version, we recommend to always review update documentation or notes provided by OpenSearch.
+
+## Can I roll back to Elasticsearch if I face issues after migrating to OpenSearch?
+
+While rolling back is generally not recommended, if you experience any significant issues after the migration, a rollback to Elasticsearch 7 is still possible. The rollback may require downtime and data migration, but we will facilitate this process.
+
+Because long-term support, security patches and updates will be provided for OpenSearch, a rollback should be a temporary solution. After the compatibility issues are fixed, to ensure the project's security and long-term support, it should be migrated to OpenSearch.
```

Comment:
```suggestion
Because long-term support, security patches, and updates will be provided for OpenSearch, a rollback should be a temporary solution. After the compatibility issues are fixed, to ensure the project's security and long-term support, it should be migrated to OpenSearch.
```